### ISSUE TITLE
V56 Breaks these/many addons that worked on v55

### ISSUE THREAD

**Issue by glen777** [BrowserWorks/waterfox#525]
A late post:    when Waterfox updated to version56 some months ago, it broke many of my most needed addons, including .maff (Mozilla Archive Format File), 'Copy All Urls', 'Quote Url Text', etc.  So I went back to v55, and blocked all updating.   For many of us, the addons are the most important part of the browser (which is why we had to leave Firfox).  I really appreciate all the hard coding work done on Waterfox, but it will be impossible for me to use Waterfox beyond v55 until these are fixed.   Anyway, I do applaud your work so far.

**Comment by laniakea64** [comment]
Did you try disabling e10s in Waterfox 56?

Waterfox Preferences > General, un-check "Enable multi-process Waterfox".  Restart Waterfox when prompted.

**Comment by glen777** [comment]
Ok, thank you.  I will download  the portable 56 and try this.

**Comment by grahamperrin** [comment]
## Mozilla Archive Format

Cross-reference https://redd.it/84fe80

Can you describe the breakage? 

Compatibility reports: https://addons.mozilla.org/compatibility/reporter/%7B7f57cf46-4467-4c2d-adfa-0cba7c507e54%7D

## CopyAllURLs

Compatibility reports: https://addons.mozilla.org/compatibility/reporter/%7B960BE052-4847-422b-9AD6-8631D3D0A607%7D

## QuoteURLText

Compatibility reports: https://addons.mozilla.org/compatibility/reporter/quoteurltext%40jay.palat

**Comment by MrAlex94** [comment]
If you have add-ons that don't work, then generally disabling e10s solves the issue.

**Comment by MrAlex94** [comment]
Any updates, without e10s do you still have the issues?

**Comment by ilu33** [comment]
Save-to-file is broken in Waterfox 56 although its still working in FF 56. Disabling e10s made it work in WF. Can't say anything about the extensions the TO is using though.

**Comment by grahamperrin** [comment]
> Save-to-file …

Do you have a link for that extension? Thanks.

**Comment by grahamperrin** [comment]
@ilu33 there's an issue for _Save File to_, also known as _SaveFileTo_. Is _Save-to-file_ another 'known as' for the same extension?

(The name implies something different, but there's no match for _Save-to-file_ on page 1 of results at <https://www.ecosia.org/search?q=%22Save-to-file%22+site%3Aaddons.mozilla.org&ref=waterfox>.).

**Comment by ilu33** [comment]
Its this one: https://bitbucket.org/atte-kemppila/save-file-to/wiki/Home or here: https://addons.mozilla.org/en-US/firefox/addon/save-file-to - you see the dashes there? ;-P
Very useful addon - another one of those little features that should really be built into the browser - saving a lot of downloads/PDFs/pictures is a pita without the addon. And, as I said, saved by disabling e10s.

**Comment by laniakea64** [comment]
@ilu33 https://github.com/MrAlex94/Waterfox/issues/476

**Comment by grahamperrin** [comment]
> … you see the dashes there? ;-P

Yep, but the three words are reversed :-) and I didn't want to jump to conclusions. Thanks.

**Comment by glen777** [comment]
OP here. I've been using the older Waterfox 55, but today tried the 56 again. Update:
— Maff Moz Archive format:  they are discontinuing this.   But there is something of similar idea called 'Save Page WE'; I haven't tried it.  All the 1000s of my saved .maff files would need to be converted before they could be opened in newer fx/Wf.
—QuoteURLText: A new version 2 WORKS!, even with e10 enabled
— copyallurls:  [ failed. Gave error" 'webnav is undefined'; failed even when disabled e10s.   ].   no longer being maintained due to signage.
      similar to:
— Copy All Tabs by Charles  [BUT, also failed to work; failed even when disabled e10s ]
—-

**Comment by grahamperrin** [comment]
[Legacy CopyAllURLs compatibility with Waterfox 56.2.0 : waterfox](https://new.reddit.com/r/waterfox/comments/8km6kg/legacy_copyallurls_compatibility_with_waterfox/) – the extension works as expected, please join the discussion there if you need help. Thanks.

**Comment by glen777** [comment]
@ grahamperrin:
No, this was not my experience at all.   CopyAllURLs worked completely fine for years on both firefox and waterfox.   But when I upgraded Waterfox to v56, it stopped working completely.  NO other changes or 'preferencess' were made at all to Waterfox! 
I even then tried a fresh install of waterfox, and the same thing happened.   Even disabling 'e10' failed to get it to work.  
It's not helpful to say "preferences must be set properly", and then not mention what these might be.   In any case, NO changes were made to Waterfox when I upgraded it to 56, — but CopyAllURLs just quit working!  And the same thing on a fresh, new install of waterfox 56!  So it's clear, that CopyAllURLs is just not compatible with v56. (using Windows 7).
